CMakeLists.txt

```diff
@@ -40,6 +42,14 @@ add_library(rtcdcpp SHARED
         ${LIB_HEADERS}
         ${LIB_SOURCES})
 
+if (SPDLOG_FOUND)
```

Comment:
Ok, the new updated review now addresses this. We will fine tune as we go, I am still not a 100% pleased with this, but it's getting better. I chose spdlog after quite a bit of vetting .... it's a really cool library ..... except now, I've just realize it does not have a "configuration from file feature"... but there is a [open request for it](https://github.com/gabime/spdlog/issues/162) so let's hope soon.